src/sourmash/tax/tax_utils.py

```diff
@@ -428,6 +429,70 @@ def write_summary(summarized_gather, csv_fp, *, sep=',', limit_float_decimals=Fa
             w.writerow(rD)
 
 
+def write_human_summary(summarized_gather, out_fp, display_rank):
+    '''
+    Write human-readable taxonomy-summarized gather results for each rank.
+    '''
+    # query_name,fraction,lineage,f_weighted_at_rank,query_ani_at_rank
+    # how do we get query_ani_at_rank to be nonzero?
+    header = SummarizedGatherResult._fields
+
+    found_ANI = False
+    results = [] 
+    for rank, rank_results in summarized_gather.items():
+        if rank == display_rank:
```

Comment:
ah, i see - can classify at any rank, but print classifications only if available at given rank - yes?

This might be something we want to allow for `krona` output, too